ERROR - 2025-10-07 11:58:57 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lms_provido', 3306, '', 0)
#1 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\ITE311-PROVIDO\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROVIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\ITE311-PROVIDO\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\ITE311-PROVIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2025-10-07 12:00:21 --> mysqli_sql_exception: Table 'users' already exists in C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\ITE311-PROVIDO\app\Database\Migrations\2025-09-04-055231_CreateUsersTable.php(40): CodeIgniter\Database\Forge->createTable('users')
#5 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateUsersTable->up()
#6 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROVIDO\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROVIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-07 12:06:34 --> mysqli_sql_exception: Table 'courses' already exists in C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\ITE311-PROVIDO\app\Database\Migrations\2025-09-04-055232_CreateCoursesTable.php(31): CodeIgniter\Database\Forge->createTable('courses')
#5 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateCoursesTable->up()
#6 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROVIDO\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROVIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-07 12:08:11 --> mysqli_sql_exception: Table 'courses' already exists in C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\ITE311-PROVIDO\app\Database\Migrations\2025-09-04-055232_CreateCoursesTable.php(31): CodeIgniter\Database\Forge->createTable('courses')
#5 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateCoursesTable->up()
#6 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROVIDO\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROVIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-07 12:08:38 --> mysqli_sql_exception: Table 'lessons' already exists in C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `l...', 0)
#1 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `l...')
#2 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `l...')
#3 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `l...')
#4 C:\xampp\htdocs\ITE311-PROVIDO\app\Database\Migrations\2025-09-04-055234_CreateLessonsTable.php(37): CodeIgniter\Database\Forge->createTable('lessons')
#5 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateLessonsTable->up()
#6 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROVIDO\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROVIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-07 12:09:23 --> mysqli_sql_exception: Table 'enrollments' already exists in C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROVIDO\app\Database\Migrations\2025-10-07-115511_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROVIDO\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROVIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-07 12:09:59 --> mysqli_sql_exception: Table 'enrollments' already exists in C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROVIDO\app\Database\Migrations\2025-10-07-115511_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROVIDO\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROVIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-07 12:10:55 --> mysqli_sql_exception: Table 'enrollments' already exists in C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-PROVIDO\app\Database\Migrations\2025-10-07-115511_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-PROVIDO\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-PROVIDO\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-PROVIDO\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-PROVIDO\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-PROVIDO\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
